ghci/containers/ghci-koji/Dockerfile

```
@@ -0,0 +1,17 @@
+FROM registry.fedoraproject.org/fedora:32
```

```suggestion
FROM docker.io/library/fedora:latest
```

Is there a reason you use the fedoraproject registry? This means we rely on their servers being accessible. If the content is the same as on `docker.io`, I would prefer the latter just for reliability reasons.

I would also suggest using `latest`. It points to the last *stable-release* and I think we should at least try the latest release on every build we attempt. We do not automatically deploy new images, so I think there is no harm in using `latest`. But if you want `32`, I am fine with that as well.

---

I thought the images on Fedora registry are updated regularly but it seems I was wrong. I changed it to use docker hub. I also changed the version to latest as you suggested.

FYI the latest tag sadly still points at f31 but whatever.